packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -3,6 +3,8 @@
 
 import { concatStyleSets, IStyle, mergeStyles, Stack } from '@fluentui/react';
 import React, { useCallback, useMemo, useRef } from 'react';
+/* @conditional-compile-remove(pinned-participants) */
+import { useState } from 'react';
```

Comment:
Since React is already imported. You can use `React.useState` directly to get rid of this conditional import